modules/simple_forms_api/app/services/simple_forms_api/notification_email.rb

```diff
@@ -140,6 +138,15 @@ def send_email_now(template_id, data)
       end
     end
 
+    def get_first_name
+      if user_account
+        mpi_profile = MPI::Service.new.find_profile_by_identifier(identifier_type: 'ICN', identifier: user_account.icn)
+        mpi_profile.first_name
```

Comment:
What if mpi_profile is nil here?

Reply:
I've updated the PR to account for this. I see elsewhere that we can raise `mpi_profile.error` so I'm mimicking that here, in addition to checking for `nil`.